ee/clickhouse/models/test/test_action.py

```diff
@@ -23,12 +23,12 @@ def _create_event(**kwargs) -> Event:
 
 
 def query_action(action: Action) -> Optional[List]:
-    formatted_query, params = format_action_filter(action, "")
+    formatted_query, params = format_action_filter(team_id=action.team.id, action=action, prepend="")
```

Comment:
Never do `thing.another.id` in an ORM when you could do `thing.another_id` - otherwise you're introducing a subquery where one isn't needed.

Use `action.team_id` here.